ballista/core/Cargo.toml

```diff
@@ -45,6 +45,7 @@ simd = ["datafusion/simd"]
 
 [dependencies]
 ahash = { version = "0.8", default-features = false }
+anyhow = "1"
```

Comment:
why add this dep

Reply:
used it during development, forgot to remove